2019/09/19 10:30:28 [error] 10#10: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/19 10:31:04 [error] 10#10: *3 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/20 19:45:12 [error] 6#6: *13 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: Whitespace are not supported before the value in "/var/www/symfony/.env" at line 18.
...n vars ###\nAPP_NAME= Water CMS\n###< Appl...
                       ^ line 18 offset 794 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(257): Symfony\Component\Dotenv\Dotenv->createFormatException('Whitespace are ...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Component\Doten" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/20 19:45:23 [error] 6#6: *13 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...#\nAPP_NAME=Water CMS\n###< Application va...
                       ^ line 18 offset 803 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Compon" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/21 11:45:42 [crit] 6#6: *98 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev", referrer: "http://app.dev/login"
2019/09/29 11:42:33 [error] 6#6: *21 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...AME=Application name\n###< Application va...
                      ^ line 18 offset 810 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Componen" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev", referrer: "http://app.dev/"
2019/09/29 11:42:40 [error] 6#6: *21 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...AME=Application name\n###< Application va...
                      ^ line 18 offset 810 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Componen" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET /login HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/29 11:42:47 [error] 6#6: *24 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...AME=Application name\n###< Application va...
                      ^ line 18 offset 810 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Componen" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET /login HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/29 11:42:47 [error] 6#6: *24 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...AME=Application name\n###< Application va...
                      ^ line 18 offset 810 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Componen" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/09/29 11:42:48 [error] 6#6: *21 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Symfony\Component\Dotenv\Exception\FormatException: A value containing spaces must be surrounded by quotes in "/var/www/symfony/.env" at line 18.
...AME=Application name\n###< Application va...
                      ^ line 18 offset 810 in /var/www/symfony/vendor/symfony/dotenv/Dotenv.php:477
Stack trace:
#0 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(326): Symfony\Component\Dotenv\Dotenv->createFormatException('A value contain...')
#1 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(200): Symfony\Component\Dotenv\Dotenv->lexValue()
#2 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(487): Symfony\Component\Dotenv\Dotenv->parse('# In all enviro...', '/var/www/symfon...')
#3 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(65): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array)
#4 /var/www/symfony/vendor/symfony/dotenv/Dotenv.php(85): Symfony\Component\Dotenv\Dotenv->load('/var/www/symfon...')
#5 /var/www/symfony/config/bootstrap.php(17): Symfony\Componen" while reading response header from upstream, client: 172.19.0.1, server: app.dev, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.4:9001", host: "app.dev"
2019/10/29 14:48:19 [error] 6#6: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.31.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.31.0.5:9001", host: "symfony.localhost"
2019/10/29 14:48:26 [error] 6#6: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.31.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.31.0.5:9001", host: "symfony.localhost"
2019/10/29 14:48:33 [error] 6#6: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.31.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.31.0.5:9001", host: "symfony.localhost"
2019/10/29 14:49:19 [error] 6#6: *5 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 172.31.0.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.31.0.5:9001", host: "symfony.localhost"
2019/10/29 14:59:50 [error] 9#9: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.16.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.16.5:9001", host: "symfony.localhost"
2019/10/29 14:59:54 [error] 9#9: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.16.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.16.5:9001", host: "symfony.localhost"
2019/10/29 15:31:07 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.32.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.32.5:9001", host: "symfony.localhost"
2019/10/29 15:31:07 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.32.1, server: symfony.localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://192.168.32.5:9001", host: "symfony.localhost"
2019/10/29 15:31:11 [error] 7#7: *1 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.32.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.32.5:9001", host: "symfony.localhost"
2019/10/29 15:33:39 [error] 7#7: *6 FastCGI sent in stderr: "Primary script unknown" while reading response header from upstream, client: 192.168.32.1, server: symfony.localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://192.168.32.5:9001", host: "symfony.localhost"
2019/10/30 15:30:13 [crit] 6#6: *108 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.22.0.1, server: symfony.localhost, request: "GET /client/new HTTP/1.1", upstream: "fastcgi://172.22.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/"
2019/10/30 20:57:14 [crit] 6#6: *242 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.22.0.1, server: symfony.localhost, request: "GET /client/ HTTP/1.1", upstream: "fastcgi://172.22.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/1"
2019/10/30 22:25:31 [crit] 6#6: *289 open() "/var/tmp/nginx/fastcgi/4/00/0000000004" failed (13: Permission denied) while reading upstream, client: 172.22.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.22.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/10/31 11:03:37 [crit] 7#7: *87 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/"
2019/10/31 11:20:23 [crit] 7#7: *137 open() "/var/tmp/nginx/fastcgi/2/00/0000000002" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/"
2019/10/31 12:20:20 [crit] 7#7: *329 open() "/var/tmp/nginx/fastcgi/4/00/0000000004" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/10/31 12:21:05 [crit] 7#7: *335 open() "/var/tmp/nginx/fastcgi/5/00/0000000005" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/10/31 12:25:23 [crit] 7#7: *346 open() "/var/tmp/nginx/fastcgi/6/00/0000000006" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/10/31 12:30:34 [crit] 7#7: *356 open() "/var/tmp/nginx/fastcgi/7/00/0000000007" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/10/31 12:31:09 [crit] 7#7: *358 open() "/var/tmp/nginx/fastcgi/8/00/0000000008" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/client/new"
2019/11/03 15:49:38 [crit] 7#7: *72 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/03 15:54:30 [crit] 7#7: *104 open() "/var/tmp/nginx/fastcgi/2/00/0000000002" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/"
2019/11/03 19:21:13 [error] 7#7: *153 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot declare class App\Form\ClientType, because the name is already in use in /var/www/symfony/src/Form/Client/ClientType.php on line 0
PHP message: PHP Stack trace:
PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0
PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25
PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:193
PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:133
PHP message: PHP   5. Symfony\Component\DependencyInjection\ContainerBuilder->compile() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:551
PHP message: PHP   6. Symfony\Component\DependencyInjection\Compiler\Compiler->compile() /var/www/symfony/vendor/symfony/dependency-injection/ContainerBuilder.php:754
PHP message: PHP   7. Symfony\Component\DependencyInjection\Compiler\ResolveInstanceofConditionalsPass->process() /var/www/symfony/vendor/symfony/dependency-injection/Compiler/Compiler.php:94
PHP message: PHP   8. Symfony\Component\DependencyInjection\Compiler\ResolveInstanceofConditionalsPass->processDefinition() /var/www/symfony/vendor/symfony/dependency-injection/Compiler/ResolveInstanceofConditionalsPass.php:43
PHP message: PHP   9. Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass() /var/www/symfony/vendor/symfony/dependency-injection/Compiler/ResolveInstanceofConditionalsPass.php:68
PHP message: PHP  10. Symfony\Component\Config\Resource\ClassExistenceResource->isFresh() /var/www/symfony/vendor/symfony/dependency-injection/ContainerBuilder.php:353
PHP message: PHP  11. class_exists() /var/www/symfony/vendor/symfony/config/Resource/ClassExistenceResource.php:78
PHP message: PHP  12. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ClassExistenceResource.php:78
PHP message: PHP  13. Symfony\Component\Debug\DebugClassLoader->loadClass() /var/www/
2019/11/03 19:45:59 [crit] 7#7: *183 open() "/var/tmp/nginx/fastcgi/3/00/0000000003" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/"
2019/11/03 19:46:50 [crit] 7#7: *189 open() "/var/tmp/nginx/fastcgi/4/00/0000000004" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/"
2019/11/03 19:59:23 [crit] 7#7: *195 open() "/var/tmp/nginx/fastcgi/5/00/0000000005" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/"
2019/11/03 19:59:48 [crit] 7#7: *198 open() "/var/tmp/nginx/fastcgi/6/00/0000000006" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/users/"
2019/11/03 20:22:49 [crit] 7#7: *265 open() "/var/tmp/nginx/fastcgi/0/01/0000000010" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/new"
2019/11/03 20:52:01 [crit] 7#7: *291 open() "/var/tmp/nginx/fastcgi/3/01/0000000013" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "POST /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/new"
2019/11/03 21:05:32 [crit] 7#7: *304 open() "/var/tmp/nginx/fastcgi/6/01/0000000016" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/new"
2019/11/03 21:18:55 [crit] 7#7: *360 open() "/var/tmp/nginx/fastcgi/8/01/0000000018" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/2/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:19:49 [crit] 7#7: *363 open() "/var/tmp/nginx/fastcgi/9/01/0000000019" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/2/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:25:29 [crit] 7#7: *379 open() "/var/tmp/nginx/fastcgi/0/02/0000000020" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/2/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:39:32 [crit] 7#7: *405 open() "/var/tmp/nginx/fastcgi/1/02/0000000021" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/3/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:43:03 [crit] 7#7: *418 open() "/var/tmp/nginx/fastcgi/2/02/0000000022" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/3/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:46:03 [crit] 7#7: *429 open() "/var/tmp/nginx/fastcgi/3/02/0000000023" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/3/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/03 21:48:29 [crit] 7#7: *435 open() "/var/tmp/nginx/fastcgi/4/02/0000000024" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/3/edit"
2019/11/03 21:49:52 [crit] 7#7: *445 open() "/var/tmp/nginx/fastcgi/5/02/0000000025" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/03 21:50:26 [crit] 7#7: *448 open() "/var/tmp/nginx/fastcgi/6/02/0000000026" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/"
2019/11/03 21:52:33 [crit] 7#7: *457 open() "/var/tmp/nginx/fastcgi/7/02/0000000027" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/03 21:54:02 [crit] 7#7: *476 open() "/var/tmp/nginx/fastcgi/8/02/0000000028" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/ HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/04 00:53:28 [crit] 7#7: *560 open() "/var/tmp/nginx/fastcgi/9/02/0000000029" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/?client_finder%5Bcin%5D=&client_finder%5BfullName%5D=&client_finder%5BphoneNumber%5D= HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/?client_finder%5Bcin%5D=fa0000&client_finder%5BfullName%5D=&client_finder%5BphoneNumber%5D="
2019/11/04 01:10:42 [crit] 7#7: *577 open() "/var/tmp/nginx/fastcgi/0/03/0000000030" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/?client_finder%5Bcin%5D=&client_finder%5BfullName%5D=&client_finder%5BphoneNumber%5D="
2019/11/04 01:15:42 [error] 7#7: *579 FastCGI sent in stderr: "PHP message: PHP Notice:  Array to string conversion in /var/www/symfony/src/Form/Client/ClientType.php on line 23
PHP message: PHP Stack trace:
PHP message: PHP   1. {main}() /var/www/symfony/public/index.php:0
PHP message: PHP   2. App\Kernel->handle() /var/www/symfony/public/index.php:25
PHP message: PHP   3. App\Kernel->boot() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:193
PHP message: PHP   4. App\Kernel->initializeContainer() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:133
PHP message: PHP   5. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/http-kernel/Kernel.php:486
PHP message: PHP   6. Symfony\Component\Config\ConfigCache->isFresh() /var/www/symfony/vendor/symfony/config/ConfigCache.php:60
PHP message: PHP   7. Symfony\Component\Config\Resource\SelfCheckingResourceChecker->isFresh() /var/www/symfony/vendor/symfony/config/ResourceCheckerConfigCache.php:99
PHP message: PHP   8. Symfony\Component\Config\Resource\ReflectionClassResource->isFresh() /var/www/symfony/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php:34
PHP message: PHP   9. Symfony\Component\Config\Resource\ReflectionClassResource->computeHash() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:52
PHP message: PHP  10. ReflectionClass->__construct() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:105
PHP message: PHP  11. spl_autoload_call() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:105
PHP message: PHP  12. Symfony\Component\Debug\DebugClassLoader->loadClass() /var/www/symfony/vendor/symfony/config/Resource/ReflectionClassResource.php:105" while reading response header from upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/?client_finder%5Bcin%5D=&client_finder%5BfullName%5D=&
2019/11/04 01:15:42 [crit] 7#7: *579 open() "/var/tmp/nginx/fastcgi/1/03/0000000031" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/?client_finder%5Bcin%5D=&client_finder%5BfullName%5D=&client_finder%5BphoneNumber%5D="
2019/11/04 01:34:05 [crit] 7#7: *624 open() "/var/tmp/nginx/fastcgi/2/03/0000000032" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/04 01:34:33 [crit] 7#7: *627 open() "/var/tmp/nginx/fastcgi/3/03/0000000033" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/04 01:38:44 [crit] 7#7: *641 open() "/var/tmp/nginx/fastcgi/4/03/0000000034" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/04 01:41:18 [crit] 7#7: *643 open() "/var/tmp/nginx/fastcgi/5/03/0000000035" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/new HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost"
2019/11/04 10:46:40 [crit] 7#7: *71 open() "/var/tmp/nginx/fastcgi/1/00/0000000001" failed (13: Permission denied) while reading upstream, client: 172.20.0.1, server: symfony.localhost, request: "GET /Client/3/edit HTTP/1.1", upstream: "fastcgi://172.20.0.5:9001", host: "symfony.localhost", referrer: "http://symfony.localhost/Client/"
2019/11/04 11:31:55 [error] 7#7: *325 client intended to send too large body: 4307799 bytes, client: 172.20.0.1, server: symfony.localhost, request: "POST /users/1/edit HTTP/1.1", host: "symfony.localhost", referrer: "http://symfony.localhost/users/1/edit"
2019/11/04 11:32:20 [error] 7#7: *330 client intended to send too large body: 2061194 bytes, client: 172.20.0.1, server: symfony.localhost, request: "POST /users/1/edit HTTP/1.1", host: "symfony.localhost", referrer: "http://symfony.localhost/users/1/edit"
